tests/KeyDBTest.php

```diff
@@ -32,15 +31,12 @@ public function testCreateFromRootMetadata(): void
         $keyDb = KeyDB::createFromRootMetadata($rootMetadata);
         self::assertInstanceOf(KeyDB::class, $keyDb);
         // Get the first key for comparison.
-        $key = $rootMetadata->getKeys()->getIterator()->current();
-        $jsonEncodedKey = json_encode($key);
-        // Re-create the the key id which can be used by getKey().
-        $keyNormalized = JsonNormalizer::asNormalizedJson($key);
-        $keyId256 = hash('sha256', $keyNormalized);
-        self::assertSame($jsonEncodedKey, json_encode($keyDb->getKey($keyId256)));
-
-        // Ensure that changing a value in the key does not affect the internal state of the KeyDB object.
-        $key256['keyval']['new_key'] = 'new_value';
-        self::assertSame($jsonEncodedKey, json_encode($keyDb->getKey($keyId256)));
+        $keys = $rootMetadata->getKeys();
+        $key = array_shift($keys);
```

Comment:
```suggestion
        $key = reset($keys);
```

Nit: it might be wise to avoid state mutation where possible.